src/test/java/seedu/address/model/UserPrefsTest.java

```diff
@@ -1,21 +1,41 @@
 package seedu.address.model;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static seedu.address.testutil.Assert.assertThrows;
 
+import java.util.Objects;
+
 import org.junit.jupiter.api.Test;
 
 public class UserPrefsTest {
 
+    private static final UserPrefs USER_PREFS = new UserPrefs();
+
     @Test
     public void setGuiSettings_nullGuiSettings_throwsNullPointerException() {
-        UserPrefs userPref = new UserPrefs();
-        assertThrows(NullPointerException.class, () -> userPref.setGuiSettings(null));
+        assertThrows(NullPointerException.class, () -> USER_PREFS.setGuiSettings(null));
     }
 
     @Test
     public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
-        UserPrefs userPrefs = new UserPrefs();
-        assertThrows(NullPointerException.class, () -> userPrefs.setEduMateFilePath(null));
+        assertThrows(NullPointerException.class, () -> USER_PREFS.setEduMateFilePath(null));
+    }
+
+    @Test
+    public void hashCode_validUserPrefs_success() {
+        assertEquals(USER_PREFS.hashCode(), Objects.hash(USER_PREFS.getGuiSettings(),
+                USER_PREFS.getEduMateFilePath()));
+    }
+
+    @Test
+    public void equals_sameObject_true() {
```

Comment:
Same clarification as the first one.

Reply:
Same object check is required by Jacoco.